notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb

```diff
@@ -0,0 +1,397 @@
+    "- *astropy.io fits* for accessing FITS files\n",
+    "- *astropy.wcs WCS* to interpret the World Coordinate Systems\n",
+    "- *matplotlib.pyplot* for plotting the data\n",
+    "- *astroquery.mast* to do the catalog search. \n",
+    "- *zipfile* to unzip the downloaded file\n",
+    "\n",
+    "For ease of use later on, we also set a the root url for our requests. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from astropy.io import fits\n",
+    "import requests\n",
+    "from astroquery.mast import Catalogs\n",
+    "import zipfile\n",
+    "from astropy.wcs import WCS\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "###CHANGE THE NEXT LINE in Pull Request ###\n",
+    "#This is the url we are using.\n",
+    "urlroot = \"https://mastdev.stsci.edu/tesscut/api/v0.1\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# DELETE THIS CELL at time of Pull Request\n",
+    "\n",
+    "#Next line puts your mast api token in a useful variable for later.\n",
+    "#See https://auth.mast.stsci.edu/info\n",
+    "masttoken = %env MAST_DV_TOKEN"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"catalog_ID\"></a>\n",
+    "## Get RA and Dec of your Target Using Astroquery Catalogs\n",
+    "\n",
+    "Here we use the astroquery.mast Catalogs query_criteria function to request the RA and Dec of the object with TIC ID = 261136679. This happens to be Pi Mensae. We collect the Right Ascension and Declination and store them in their own variables.  \n",
+    "<p>\n",
+    "Here we do a cone search using Catalogs.query_object on the TIC catalog. The advantage of doing this is that it gives us the nearby stars as well as the star we are looking for. The resulting table is sorted by distance from the requested object. We print out the ID and a few other TIC quantities to ensure we found the star we were looking for. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ticid = 261136679\n",
+    "\n",
+    "starName = \"TIC \" + str(ticid)\n",
+    "rad_search = 5 / 60 #radius in degrees\n",
+    "\n",
+    "catalogData = Catalogs.query_object(starName, radius = rad_search, catalog = \"TIC\")\n",
+    "Ra = catalogData[0]['ra']\n",
+    "Dec = catalogData[0]['dec']\n",
+    "\n",
+    "#Print out the first row in the table\n",
+    "print( catalogData[:5]['ID', 'Tmag', 'Jmag', 'ra', 'dec', 'objType'] )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Create a list of nearby bright stars (tess magnitude less than 14) from the rest of the data for later.\n",
+    "bright=catalogData['Tmag'] < 14\n",
```

Comment:
Add space after #, and around = . You've mostly done this throughout, but there are a few places that are still missing spaces.